.atomist/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Looking at the code, it appears smart enough to do the right thing for things that properly start with `I`, so we'll go with @kipz initial suggestion.